boards/nxp/imx91_evk/imx91_evk_mimx9131_defconfig

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# ARM Options
+CONFIG_AARCH64_IMAGE_HEADER=y
+CONFIG_ARMV8_A_NS=y
+
+# MMU Options
+CONFIG_MAX_XLAT_TABLES=64
```

Comment:
Let's use default value "8" if it is enough,  or change to a samller one, "64" is too large,  each table uses about 4k memory space, so keep it as small as enough.

Reply:
The SoC `mmu_region[]` has 7 fixed entries currently, will reduce it to 24 to leave some space for drivers's dynamic MMIO maps.

Reply:
Actually one table has 512 entries.

Reply:
what's the relation between the entries and the mmu_region[]?

Reply:
The help info says: "Number of translation tables required is decided based on how many discrete memory regions (both normal and device memory) are present on given platform and how much granularity is required while assigning attributes to these memory regions."
Currntly, the least working value of CONFIG_MAX_XLAT_TABLES is 7, so, we'd better give some space for coming drivers' MMIO mappings, say 24?

Reply:
okay for me, thanks.